Source/Core/Core/PowerPC/RSO.cpp

```diff
@@ -0,0 +1,642 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "RSO.h"
```

Comment:
This should alternatively probably not be within PowerPC, since it's not really related to actual PPC emulation. `Core/Debugger` is probably more suitable or something.